components/InsectSelector.tsx

```diff
@@ -25,27 +25,34 @@ const InsectSelector = ({ playerHand: insects }: InsectSelectorProps) => {
     <div
       className={'flex flex-row justify-center content-center h-full flex-wrap'}
     >
-      {individualInsects.map((insectName, i) => (
-        <div
-          className={classNames('insect', {
-            insect__selected:
-              gameState.context.selectedUnplayedInsect! === insectName,
-            insect__disabled: !playerToMove,
-          })}
-          key={i + 1}
-          onClick={() => {
-            playerToMove && sendToGame('UNPLAYEDPIECE.SELECT', { insectName })
-          }}
-        >
-          <Image
-            src={`/icons/${insectName.toLowerCase()}.svg`}
-            width="50"
-            height="50"
-            alt={`${insectName}`}
-            priority
-          />
-        </div>
-      ))}
+      {individualInsects.map((insectName, i) => {
+        const insectCanBePlayed = gameState.context.insectsAllowedToPlace?.includes(
+          insectName
+        )
+        const isSelectedInsect =
+          gameState.context.selectedUnplayedInsect! === insectName
+        return (
+          <div
+            className={classNames('insect', {
+              insect__selected: isSelectedInsect,
+              insect__disabled: !playerToMove || !insectCanBePlayed,
+            })}
+            key={i + 1}
+            onClick={() => {
+              ;(playerToMove || !insectCanBePlayed) &&
+                sendToGame('UNPLAYEDPIECE.SELECT', { insectName })
+            }}
+          >
+            <Image
+              src={`/icons/${insectName.toLowerCase()}.svg`}
+              width="50"
+              height="50"
```

Comment:
Is this responsive?

Reply:
I don't think I actually wrote this but moved it from somewhere (so it seems like I contributed it). 
![image](https://user-images.githubusercontent.com/6740889/100283591-175fe280-2f6e-11eb-86bc-ee7b05edd680.png)
Seems reasonable and with Cedric contribution, should be reduced to even smaller footprint. But agree that it's not with responsiveness in mind and could be improved. Let me know what you think.

Reply:
Ok, kan eventueel in andere PR opgelost worden